DepositUI/Pages/Deposit.razor

```diff
@@ -1,10 +1,6 @@
 ﻿@page "/deposit"
-
 @using DepositUI.Data
 @using DepositUI.Components
-@using System.Text.Json
-@using System.Text.Json.Serialization;
-@using System.Net.Http.Json;
 @inject IHttpClientFactory ClientFactory;
 
 <h1>Deposit Calculations</h1>
```

Comment:
Consider using the Validation Form instead of just _input_ tag. This is able to use input form for numbers and remove _NumberInputCheck_ method and field checks. For details see here: https://docs.microsoft.com/ru-ru/aspnet/core/blazor/forms-validation?view=aspnetcore-5.0